trends/trends.py

```diff
@@ -54,22 +56,49 @@ async def wrapper(*args, **kwargs):
     return wrapper
 
 
+def separate_emoji_and_text(query):
```

Comment:
uses the name of the emoji in the search instead of the emoji itself. I actually thought it was doing semantic search instead of fulltext search, so I don't know how useful this is (other then the fact that emojis no longer break the search.)

I'm working on a different version that uses Chroma as a vector storage to do semantic search as well.